app.js

```diff
@@ -1,15 +1,24 @@
 const express = require("express");
-const { getCategoriesC } = require("./controllers/controllers");
+const { getCategoriesC, getReviewsByIdC } = require("./controllers/controllers");
 const error_404_msg = "Sorry can't find that!";
+const error_400_msg = "invalid type please check your input"
+
 const app = express();
 app.use(express.json());
 
 app.get("/api/categories", getCategoriesC);
+app.get("/api/reviews/:review_id", getReviewsByIdC);
 
 app.use((err, req, res, next) => {
+  if (err.code === "22P02") {
```

Comment:
If you wanted to, you could separate the PSQL errors and custom errors into their own error handling middleware functions like we do with each endpoint. Furthermore, we could also separate the error handling into it's own directory.